ntoskrnl/cc/view.c

```diff
@@ -411,8 +412,7 @@ CcRosFlushDirtyPages (
         if (--SharedCacheMap->OpenCount == 0)
             CcRosDeleteFileCache(SharedCacheMap->FileObject, SharedCacheMap, &OldIrql);
 
-        if (!NT_SUCCESS(Status) && (Status != STATUS_END_OF_FILE) &&
-            (Status != STATUS_MEDIA_WRITE_PROTECTED))
```

Comment:
These checks that you call "obsolete" were probably there to avoid the DPRINT in case of common failure codes.
Have you tested this? Do you see any debug spam after removing the checks?

Reply:
> avoid the DPRINT

That is what I thought at first, which would mean they would have been misplaced.

> Have you tested this?

Nothing to test here, as these statuses cannot happen since @zefklop's `CcRosFlushVacb()` change.